src/Symfony/Component/Routing/Loader/XmlFileLoader.php

```diff
@@ -229,6 +229,16 @@ private function parseConfigs(\DOMElement $node, $path)
             }
         }
 
+        if ($controller = $node->getAttribute('controller')) {
+            if (isset($defaults['_controller'])) {
+                $name = $node->hasAttribute('id') ? sprintf('"%s"', $node->getAttribute('id')) : sprintf('the %s tag', $node->tagName);
+
+                throw new \InvalidArgumentException(sprintf('The routing file "%s" must not specify both the "controller" attribute and the defaults key "_controller" for %s.', $path, $name));
```

Comment:
`Route "%s" must must not specify both the "controller" attribute and the defaults key "_controller" in file "%s"'` ?